portal/server/routes/admin/activity.js

```diff
@@ -0,0 +1,54 @@
+const express = require('express');
+const { validationErrorHandler } = require('../../helpers');
+
+const router = express.Router();
+
+router.get('/activity', async (req, res) => res.render('activity/generate-activity-report.njk', { user: req.session.user }));
+
+router.post('/activity', async (req, res) => {
+  const { usersSearchTerm } = req.body;
+
+  if (!usersSearchTerm) {
```

Comment:
We'll need to make sure that we can handle special characters like `/` and `@`, etc. in the search term, since these are valid in email addresses (our current implementation with redirecting to `/activity/${usersSearchTerm}` will have trouble with `/`)